ex2_hal/charon/daemons/nmea_daemon.c

```diff
@@ -13,7 +13,7 @@
 #include "task_manager/task_manager.h"
 #include "NMEAParser.h"
 
-#define NMEA_TASK_SIZE configMINIMAL_STACK_SIZE
+#define NMEA_TASK_SIZE 256
```

Comment:
A bit surprised the minimal stack size isn't OK

Reply:
Me too. It can probably be optimized